recipes/qt/6.x.x/conanfile.py

```diff
@@ -1012,15 +1029,18 @@ def package_info(self):
         self.runenv_info.define("QT_PLUGIN_PATH", os.path.join(self.package_folder, "plugins"))
         self.buildenv_info.define("QT_PLUGIN_PATH", os.path.join(self.package_folder, "plugins"))
 
-        self.buildenv_info.define("QT_HOST_PATH", self.package_folder)
+        # TODO
+        # qt_host_path = self.dependencies.direct_build["qt"].package_folder if self._is_mobile_os else self.package_folder
+        qt_host_path = self.package_folder
+        self.buildenv_info.define("QT_HOST_PATH", qt_host_path)
```

Comment:
Also dunno how to solve. In `package_info` this doesn't work: `self.dependencies.direct_build["qt"].package_folder` is `None`, although `self.dependencies.direct_build["qt"]` returns some kind of a valid object.